packages/blueprints-integration/src/action.ts

```diff
@@ -34,6 +34,10 @@ export interface IBlueprintActionManifestDisplay {
 	nextPieceTags?: string[]
 	/** String that can be used to identify adlibs that are equivalent to each other */
 	uniquenessId?: string
+	/** When not playing, display in the UI as playing, and vice versa. Useful for Adlibs that toggle something off when taken */
+	invertOnAirState?: boolean
+	/** Do not assign a hotkey to this adlib */
+	noHotKey?: boolean
```

Comment:
I don't think this actually does anything. I don't see anything reading from `noHotKey`.